### DISCUSSION TITLE
Creating Fat/Universal versions of libraries

### DISCUSSION THREAD

**Discussion by Vithanco** [Homebrew/discussions#399] [Everyday usage]
I have some dependencies for my Application that I fetch via home-brew. I have now a completely new problem: if I want to provide a universal software to the App Store then I will need to have my libraries in a fat/universal format. 

1) Can I somehow fetch both (x86_64 and arm64) bottled versions from the central repository? Or is this something that I could do differently, e.g. via `--build-from-source`. How would I configure the build within the home-brew ecosystem?

2) I guess I should combine them with `lipo`. Is this something that could be automated?

**Comment by SMillerDev** [comment]
Homebrew really doesn't support, nor wants to support this use case. The best way is really to build your own dependencies.

**Reply by Vithanco** [reply]
Clear answer. Got it. 
Anyone with advise on how to handle this best outside of homebrew? I guess I could still use the bottled versions...
